src/test/java/com/microsoft/aad/msal4j/AuthorityTest.java

```diff
@@ -174,4 +174,27 @@ public void testValidateAuthorityEmptyPathSegments(String authority) throws Malf
     public void testValidateAuthorityEmptyPath() throws MalformedURLException {
         Authority.validateAuthority(new URL("https://login.microsoftonline.com"));
     }
+
+    @Test
+    public void testCloundEndpointConstants() throws MalformedURLException {
+        PublicClientApplication pca = PublicClientApplication.builder("client_id")
+                .authority(AzureCloudEndpoint.AzurePublic.endpoint + "common")
+                .build();
+        Assert.assertEquals(pca.authority(), TestConfiguration.AAD_COMMON_AUTHORITY);
+
+        pca = PublicClientApplication.builder("client_id")
+                .authority(AzureCloudEndpoint.AzureChina.endpoint + "common")
+                .build();
+        Assert.assertEquals(pca.authority(), TestConfiguration.AAD_COMMON_AUTHORITY_CN);
+
+        pca = PublicClientApplication.builder("client_id")
+                .authority(AzureCloudEndpoint.AzureGermany.endpoint + "common")
+                .build();
+        Assert.assertEquals(pca.authority(), TestConfiguration.AAD_COMMON_AUTHORITY_DE);
+
+        pca = PublicClientApplication.builder("client_id")
+                .authority(AzureCloudEndpoint.AzureUsGovernment.endpoint + "common")
+                .build();
+        Assert.assertEquals(pca.authority(), TestConfiguration.AAD_COMMON_AUTHORITY_US);
+    }
```

Comment:
I do not think this test bring any value.

Reply:
I based it off a test of setting default authority, but I guess for these endpoints just setting the authority is not a meaningful test. A good test would be acquiring a token at each endpoint, but that would mainly be a test of making sure the endpoint still works rather than if the library does, so I just removed the test and related config constants altogether.